Packs/RecordedFuture/Playbooks/playbook-Recorded_Future_C2_Threat_Assessment.yml

```diff
@@ -0,0 +1,382 @@
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: modules
+                filters:
+                - - operator: containsString
+                    left:
+                      value:
+                        simple: modules.brand
+                      iscontext: true
+                    right:
+                      value:
+                        simple: Recorded Future
+                    ignorecase: true
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: modules.state
+                      iscontext: true
+                    right:
+                      value:
+                        simple: active
+                    ignorecase: true
+                accessor: brand
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": 410,
+          "y": 505
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "28":
+    id: "28"
+    taskid: dc8fb6e4-dd90-4bfb-8804-8bc42cdef87e
+    type: regular
+    task:
+      id: dc8fb6e4-dd90-4bfb-8804-8bc42cdef87e
+      version: -1
+      name: Get Threat Assessment from Recorded Future
+      description: Get an indicator of the risk based on context.
+      script: Recorded Future v2|||recordedfuture-threat-assessment
+      type: regular
+      iscommand: true
+      brand: Recorded Future v2
+    nexttasks:
+      '#none#':
+      - "24"
+    scriptarguments:
+      context:
+        simple: c2
+      cve:
+        simple: ${inputs.CVE}
+      domain:
+        simple: ${inputs.Domain}
+      file:
+        simple: ${inputs.MD5},${inputs.SHA1},${inputs.SHA256}
+      ip:
+        simple: ${inputs.IP}
+      url:
+        simple: ${inputs.URL}
+    reputationcalc: 1
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 410,
+          "y": 820
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "29":
+    id: "29"
+    taskid: 611b3988-cbe6-4bc4-8044-6f00b811d9bc
+    type: title
+    task:
+      id: 611b3988-cbe6-4bc4-8044-6f00b811d9bc
+      version: -1
+      name: Threat Assessment Using Recorded Future API
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "26"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 410,
+          "y": 370
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+view: |-
+  {
+    "linkLabelsPosition": {
+      "16_24_#default#": 0.57
+    },
+    "paper": {
+      "dimensions": {
+        "height": 1020,
+        "width": 740,
+        "x": 50,
+        "y": 50
+      }
+    }
+  }
+inputs:
```

Comment:
All the playbooks are made to be sub playbooks. I'm fine with leaving them empty (if that is the way other sub playbooks work in XSOAR). However we thought it would be a sane default to use the default contexts and pull from there.

Reply:
They're merged in the latest few commits.